R/zzz.R

```diff
@@ -7,23 +7,14 @@
 }
 
 get_train_template_id <- function() {
-  versions <- unique(CIVIS_ML_TEMPLATE_IDS$version)
-  latest <- max(versions)
-  id <- CIVIS_ML_TEMPLATE_IDS[CIVIS_ML_TEMPLATE_IDS$version == latest &
-                              CIVIS_ML_TEMPLATE_IDS$name == "train", "id"]
 
-  if (!inherits(try(scripts_list_custom(id), silent = TRUE), "try-error")) {
-    i <- 1
-    while (length(scripts_list_custom(id)) == 0 &&
-           i < length(versions)) {
-      # use the previous version; assume that only the latest version is in internal release
-      prev <- versions[length(versions) - i]
-      id <- CIVIS_ML_TEMPLATE_IDS[CIVIS_ML_TEMPLATE_IDS$version == prev &
-                                  CIVIS_ML_TEMPLATE_IDS$name == "train", "id"]
-      i <- i + 1
-    }
-  }
+  CIVIS_ML_TEMPLATE_IDS <- get_template_ids_all_versions()
```

Comment:
same comment re: var naming of `CIVIS_ML_TEMPLATE_IDS`

Reply:
yep, in can be lower case now since it's not referring to global data

Reply:
Changed to lower case. Thanks! Resolving this convo